test/cli.zig

```diff
@@ -174,4 +174,13 @@ fn testZigFmt(zig_exe: []const u8, dir_path: []const u8) !void {
     const run_result3 = try exec(dir_path, true, &[_][]const u8{ zig_exe, "fmt", dir_path });
     // both files have been formatted, nothing should change now
     testing.expect(run_result3.stdout.len == 0);
+
+    // Check UTF-16 decoding
+    const fmt4_zig_path = try fs.path.join(a, &[_][]const u8{ dir_path, "fmt4.zig" });
+    var unformatted_code_utf16 = "\xff\xfe \x00 \x00 \x00 \x00/\x00/\x00 \x00n\x00o\x00 \x00r\x00e\x00a\x00s\x00o\x00n\x00";
+    try fs.cwd().writeFile(fmt4_zig_path, unformatted_code_utf16);
```

Comment:
Thinking out loud, it seems that no test is cleaning up the temporary files it creates.

Reply:
The argument `dir_path` is a temporary directory which is removed between tests. However, there is no removal at the end.

Reply:
I'd add a `defer fs.cwd().deleteTree(dir_path) catch {}` below `dir_path` declaration and call it a day then.

Reply:
I just added this in 0becc17e6b18d9d755729ba0f471266d4b1ceab1.